src/main/java/org/assertj/core/util/DoubleComparator.java

```diff
@@ -23,7 +21,7 @@ public DoubleComparator(double epsilon) {
   }
 
   @Override
-  public int compare(Double x, Double y) {
+  protected int compareNonNull(Double x, Double y) {
```

Comment:
I actually thought we were using `complexCloseEnough` instead of `closeEnough`, I can't remember or find out why we are not doing this (facepalm), it might just be an unfinished refactoring.